plugins/woocommerce/client/admin/client/task-lists/fills/Marketing/Plugin.tsx

```diff
@@ -94,16 +121,32 @@ export const Plugin: React.FC< PluginProps > = ( {
 						{ __( 'Activate', 'woocommerce' ) }
 					</Button>
 				) }
-				{ ! isInstalled && (
+				{ ! isInstalled && ! installExternal && (
 					<Button
 						disabled={ isDisabled }
 						isBusy={ isBusy }
 						isSecondary
-						onClick={ () => installAndActivate( slug ) }
+						onClick={ () => {
+							installAndActivate( slug );
+						} }
 					>
 						{ __( 'Get started', 'woocommerce' ) }
 					</Button>
 				) }
+				{ ! isInstalled && installExternal && (
+					<Button
+						disabled={ isDisabled }
+						isBusy={ isBusy }
+						isSecondary
+						onClick={ () => {
+							if ( learnMoreLink !== '' ) {
```

Comment:
What do you think if we hide the button when the learn more link is empty? Or fallback to a generic link? Otherwise, when user clicks the button, it doesn't do anything. 🤔